.github/workflows/integration.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
         env:
           CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
         with:
-          action: codecov/codecov-action@v4
+          action: codecov/codecov-action@v4.2.0
```

Comment:
I think this would be a regression? `v4` will always pick the newest `v4` 4.3.0 right now. Is the downgrade to 4.2 the goal?